docs/pages/clients/introduction.md

```diff
@@ -10,28 +10,33 @@ import ImageSpotlight from '~/components/plugins/ImageSpotlight'
 - Quickly verify the impact of changes to your project
 - Develop iOS applications without needing a machine running MacOS
 
-<ImageSpotlight alt="Diagram that visualizes the difference between Expo Go and Custom development clients" src="/static/images/development-clients-overview.png" style={{ maxWidth: 605 }} />
+Depending on your project, you may find that you need to customize the standard runtime provided in Expo Go, either to reduce your bundle size, to use a module offered by developers in the React Native community, or even to add your own custom native code. At that point, you can create a custom development client, install it on your phone, and continue developing.
 
-Depending on your project, you may find that you need to customize the standard runtime provided in Expo Go, either to reduce your bundle size, to use a module offered by developers in the React Native community, or even to add your own custom native code. At that point, you can create a custom Development Client, install it on your phone, and continue developing.
+## How it works
+
+
+<object width="100%" height="400">
+  <param name="movie" value="https://youtube.com/embed/5c60VHxUBo8" />
+  <param name="wmode" value="transparent" />
+  <embed src="https://youtube.com/embed/5c60VHxUBo8" type="application/x-shockwave-flash" wmode="transparent" width="100%" height="400" />
+</object>
+
+`expo-dev-client` is an npm package installable in any Expo or React Native project. Once installed, any Debug builds of your application will gain an extensible debug menu and the ability to load projects from Expo CLI. Release builds of your application will not change other than the addition of a few header files.
+
+Your debug builds can be shared with anyone on your team who needs to work on or review your application. Your team can develop the JavaScript portion of your application with `expo-cli` and your custom client without waiting for your native code to build until the next time you need to upgrade, install a new module, or otherwise change the native code in your project.
 
-## What you get with a custom Expo Client
+## What you'll gain by adopting expo-dev-client
 
-### Improvements to Managed workflow with Expo Go
+### If you are using the Managed workflow with Expo Go
 
 - Add any native code your project needs
 - Develop in the same environment as your released application
-- Access to Xcode and Android Studio when you need them
+- Access to the full power of Xcode and Android Studio when you need them
```

Comment:
how so?

Reply:
you can prebuild or eject and essentially nothing about your project has changed.  debug native stuff or tweak your setting without workarounds to try to maintain your dev flow or build processes.